modules/gcp/cac-igm/cac-startup.sh.tmpl

```diff
@@ -205,10 +207,17 @@ install_cac() {
             --domain-group "${domain_group}" \
             --reg-code $PCOIP_REGISTRATION_CODE \
             --sync-interval 5 \
-            2>&1 | tee $INSTALL_LOG
+            2>&1 | tee -a $CAC_INSTALL_LOG
     fi
 }
 
+if [[ ! -f "$LOG_FILE" ]]
+then
+    mkdir -p "$(dirname $${LOG_FILE})"
+    touch "$LOG_FILE"
```

Comment:
don't think you have to touch it by default. echo should create the file and it should already be 644 mode. I think all you have to do here is to make sure the directory exist.

You could also log "Script started" just to give the timestamp of when this is run.